skyportal/models/public_pages/public_source_page.py

```diff
@@ -0,0 +1,152 @@
+__all__ = ['PublicSourcePage']
+
+import json
+import jinja2
+from ligo.skymap import plot  # noqa: F401 F811
```

Comment:
Hmmm you shouldn't need ligo skymap to plot anything here??? That's for GCN events stuff (hence the ligo)

Reply:
✅ Done.